cmd/a2o-migrate/test/integration.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+
+RT=${RT:-balena-engine}
+CONTAINERIZED=${CONTAINERIZED:-}
+PROJECT="$(dirname $(readlink -f $0))/.."
+IMAGE=${IMAGE:-balena/balena-engine:beind}
+
+container_name="balena_a2o_integration_test"
+balena_container_flags="--rm --detach --name ${container_name} --privileged -v varlibbalena:/var/lib/balena-engine -v ${PROJECT}:/src:ro -w /src"
+
+set -ex
+
+[ -n "$CONTAINERIZED" ] && {
```

Comment:
Maybe simply move it to a separate file? This part does not use any of the variables above, so it can be cleaner/easier to call a separate script that calls migration commands in the container.